samples/DotNet/Azure.Messaging.EventHubs/ManagedIdentityWebApp/SendReceive.aspx.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using Azure.Messaging.EventHubs;
+using Azure.Identity;
+using System.Threading.Tasks;
+
+// Always add app to IAM roles
+// Don't use on deployment slots but only on root
+namespace ManagedIdentityWebApp
+{
+    public partial class SendReceive : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        protected async void btnSend_Click(object sender, EventArgs e)
+        {
+            await using (EventHubProducerClient producerClient = new EventHubProducerClient($"{txtNamespace.Text}.servicebus.windows.net", txtEventHub.Text, new DefaultAzureCredential()))
+            {
+                // create a batch
+                EventDataBatch eventBatch = await producerClient.CreateBatchAsync();
+
+                // add events to the batch. only one in this case. 
+                eventBatch.TryAdd(new EventData(Encoding.UTF8.GetBytes(txtData.Text)));
+                
+                // send the batch to the event hub
+                await producerClient.SendAsync(eventBatch);
+
+                txtOutput.Text = $"{DateTime.Now} - SENT{Environment.NewLine}" + txtOutput.Text;
```

Comment:
nit:  Why use string interpolation for half of this and then concatenate the second half?

Reply:
I haven't tested this, but does this look right to you?
                txtOutput.Text = $"{DateTime.Now} - SENT{Environment.NewLine} {txtOutput.Text}";

Reply:
You've got an extra space after the new line and before the output text.   Otherwise, yup.   Looks right to me.

Reply:
Removed the space.